src/ParagraphsPastePluginBase.php

```diff
@@ -148,10 +162,19 @@ protected function setValue(array $property_path, EntityInterface $entity, $valu
       $newEntity = $this->entityReferenceSelectionManager->getSelectionHandler($fieldConfig)
         ->createNewEntity($target_type, $bundle, NULL, $this->currentUser->id());
 
+      /** @var \Drupal\Core\Entity\Display\EntityFormDisplayInterface $formDisplay */
+      $formDisplay = $this->entityDisplayRepository->getFormDisplay($entity->getEntityTypeId(), $entity->bundle());
+
+      $save = $save || (!in_array($formDisplay->getComponent($fieldName)['type'], ['inline_entity_form_simple', 'inline_entity_form_complex']));
+
       if (!empty($property_path)) {
-        $this->setValue($property_path, $newEntity, $value);
+        $this->setFieldValue($newEntity, $property_path, $value, $save);
```

Comment:
Do we want to save all entities in the recursion?

Reply:
Yes, because the first time we don't have a IEF form anymore, we have to save all sub entities